composer.json

```diff
@@ -125,6 +125,9 @@
             },
             "drupal/entity_clone": {
```

Comment:
@jhuang0427 We only need one "drupal/entity_clone" entry - you can remove the old one and just have the new one.